src/components/BusinessCaseReview/index.tsx

```diff
@@ -63,6 +71,13 @@ const BusinessCaseReview = ({ values }: BusinessCaseReviewProps) => {
             />
           </div>
         </div>
+        {grtFeedbacks && grtFeedbacks.length > 0 && (
```

Comment:
I almost want this component to decide whether or not it displays itself? But I am unsure of the best place for this code.

Reply:
I thought about that... but they grey area around it draws differently in different use cases, so I wanted that outside the component. But then we need to not draw the grey area if there are no grtFeedbacks... anyway I think this check unfortunately has to be outside the component